Invoro.WebApp/Invoro/invoro/src/lib/components/Vote/Vote.component.test.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import VoteComponent from './Vote.component';
+
+describe("VoteComponent", () => {
+    let voteComponent: ReactWrapper;
+
+    describe("Star Icon rendered when isVoted is true", () => {
+        beforeEach(() => {
+            let props = {
+                isVoted: true,
+                voteHandle: () => { },
+                unvoteHandle: () => { },
+            };
+            voteComponent = mount(<VoteComponent {...props} />);
+        });
+
+        it("isVoted true Exactly one icon is rendered", () => {
+            const icons = voteComponent.find("IconButton");
+            expect(icons.length).toBe(1);
+        });
+
+        it("Full star when isVoted is true", () => {
+            const voteIcon = voteComponent.find("VoteIconComponent");
+            expect(voteIcon.length).toBe(1);
+        });
+    });
+    describe("Star Icon rendered when isVoted is false", () => {
+        beforeEach(() => {
+            let props = {
+                isVoted: false,
+                voteHandle: () => { },
+                unvoteHandle: () => { },
+            };
+            voteComponent = mount(<VoteComponent {...props} />);
+        });
+
+        it("isVoted false Exactly one icon is rendered", () => {
+            const icons = voteComponent.find("IconButton");
+            expect(icons.length).toBe(1);
+        });
+
+        it("Empty star when isVoted is false", () => {
+            const unvoteIcon = voteComponent.find("UnvoteIconComponent");
+            expect(unvoteIcon.length).toBe(1);
+        });
+    })
+
+    describe("Star clicked", () => {
```

Comment:
Star is a design thing. It has nothing to do with the Voting logic.
The name start has to be replaced with Icon/Vote or something